File: TypeScript/19_Function/18_ArrowFunction/ArrowFunction.cs
// 화살표 함수(Arrow Function): 화살표 메서드
using System;

class ArrowFunction
{
    static void Main()
    {
        Hi(); // 안녕하세요.
        Multiply(3, 5); // 15
    }

    static void Hi() => console.log("안녕하세요.");
    static void Multiply(int a, int b) => console.log(a * b);
}

File: TypeScript/19_Function/18_ArrowFunction/ArrowFunction.ts
// 화살표 함수(Arrow Function): 화살표 메서드
//[1] 기본 함수
let f1 = function(message) {
    console.log(message);
}

//[2] 화살표 함수
let f2 = (message) => {
    console.log(message);
}

//[3] 화살표 함수로 식 본문 함수 구현
let f3 = (message) => console.log(message);

f1("Arrow Function");

f2("Lambda Expression");

f3("람다 식");
